### PR TITLE
feat: add section regional finance in landing page

### PR DESCRIPTION
**Pull request by yoslie**
## Overview
Add Section Regional Finance in Landing Page

## Preview
image

## Evidence
title: Add Section Regional Finance in Landing Page
project: Portal Jabar
participants: @yoslie @Ibwedagama @doohanas @agunghide

### CODE REVIEW

**Review by Ibwedagama — APPROVED** [review]

File: pages/index.vue [hunk 1]
```diff
@@ -36,18 +36,43 @@
...
 import { GPR_SCRIPT_STATUS } from 'static/data'
+import { formatTz } from '~/utils/date'
 
 export default {
   name: 'HomePage',
   data () {
     return {
-      gprScriptStatus: GPR_SCRIPT_STATUS.LOADING
+      gprScriptStatus: GPR_SCRIPT_STATUS.LOADING,
+      isDetailOpen: false,
```
Comment (Ibwedagama) [inline pages/index.vue:66]: minor:

hello @yoslie please rename this data property to be more specific like `isDocumentDetailOpen`

File: pages/index.vue [hunk 2]
```diff
@@ -111,6 +136,36 @@ export default {
...
+        source: item.source,
+        mimetype: item.mimetype,
+        category: item.category,
+        updatedAt: formatTz(new Date(item.updated_at), 'Asia/Jakarta', 'EEEE, dd MMMM yyyy HH:mm') + ' WIB'
+      }
+    },
+    handleCloseDetail () {
+      this.resetDocumentDetail()
+      this.isDetailOpen = false
+    },
```
Comment (Ibwedagama) [inline pages/index.vue:158]: minor:

this is also need to be more specific

Comment (yoslie) [reply pages/index.vue]: thanks, updated [here](https://github.com/jabardigitalservice/portal-jabar-frontend/pull/546/commits/2d4b7b9095706fdcfbf12bebd85b54bffb40acdb)

File: pages/index.vue (hunk 1 above)
Comment (yoslie) [reply pages/index.vue]: thanks, updated [here](https://github.com/jabardigitalservice/portal-jabar-frontend/pull/546/commits/2d4b7b9095706fdcfbf12bebd85b54bffb40acdb)

File: components/RegionalFinance/index.vue [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+      const response = await this.$axios.get('/v1/public/document-archives', { params })
+      if (response.status === 200) {
+        this.items = response.data.data
+      }
+    } catch {
+      this.items = []
+    }
+  },
+  computed: {
+  },
```
Comment (agunghide) [inline components/RegionalFinance/index.vue:172]: I think this `computed` can be deleted if it hasn't been used

**Review by agunghide — APPROVED** [review]

**Review by doohanas — APPROVED** [review]

File: components/RegionalFinance/index.vue (hunk 3 above)
Comment (yoslie) [reply components/RegionalFinance/index.vue]: My bad, thanks changes [here](https://github.com/jabardigitalservice/portal-jabar-frontend/pull/546/commits/67ddc40acca758e07ef39a2244db66251c4b04fd)

**Pull request merged**